server/main.lua

```diff
@@ -1,5 +1,17 @@
 local config = require 'config.server'
 
+---@param vehNetId number
+---@param state number
```

Comment:
Should we use a boolean or string for this? We only support two states of locked or unlocked, so taking in a number could be a more confusing API than it needs to be.

Reply:
I would prefer a string over a boolean to make things clearer.